app/api/groups/get_breadcrumbs.feature

```diff
@@ -0,0 +1,200 @@
+      | 24              | 41             | 9999-12-31 23:59:59 |
+      | 25              | 24             | 9999-12-31 23:59:59 |
+      | 26              | 4              | 9999-12-31 23:59:59 |
+      | 26              | 5              | 9999-12-31 23:59:59 |
+      | 26              | 7              | 9999-12-31 23:59:59 |
+      | 26              | 9              | 9999-12-31 23:59:59 |
+      | 26              | 10             | 9999-12-31 23:59:59 |
+      | 26              | 11             | 9999-12-31 23:59:59 |
+      | 26              | 13             | 9999-12-31 23:59:59 |
+      | 26              | 14             | 9999-12-31 23:59:59 |
+      | 26              | 16             | 9999-12-31 23:59:59 |
+      | 26              | 19             | 9999-12-31 23:59:59 |
+      | 26              | 22             | 9999-12-31 23:59:59 |
+      | 26              | 25             | 2010-01-01 00:00:00 |
+      | 26              | 27             | 9999-12-31 23:59:59 |
+      | 5               | 41             | 2010-01-01 00:00:00 |
+      | 7               | 41             | 2010-01-01 00:00:00 |
+      | 9               | 41             | 2010-01-01 00:00:00 |
+      | 10              | 41             | 2010-01-01 00:00:00 |
+      | 12              | 41             | 2010-01-01 00:00:00 |
+      | 13              | 41             | 2010-01-01 00:00:00 |
+      | 14              | 41             | 2010-01-01 00:00:00 |
+      | 15              | 41             | 2010-01-01 00:00:00 |
+      | 16              | 41             | 2010-01-01 00:00:00 |
+      | 18              | 41             | 2010-01-01 00:00:00 |
+      | 19              | 41             | 2010-01-01 00:00:00 |
+      | 21              | 41             | 2010-01-01 00:00:00 |
+      | 22              | 41             | 2010-01-01 00:00:00 |
+      | 23              | 41             | 2010-01-01 00:00:00 |
+      | 25              | 41             | 2010-01-01 00:00:00 |
+    And the groups ancestors are computed
+    And the database has the following table 'groups_ancestors':
+      | ancestor_group_id | child_group_id | expires_at          |
+      | 5                 | 41             | 2010-01-01 00:00:00 |
+      | 7                 | 41             | 2010-01-01 00:00:00 |
+      | 9                 | 41             | 2010-01-01 00:00:00 |
+      | 10                | 41             | 2010-01-01 00:00:00 |
+      | 12                | 41             | 2010-01-01 00:00:00 |
+      | 13                | 41             | 2010-01-01 00:00:00 |
+      | 14                | 41             | 2010-01-01 00:00:00 |
+      | 15                | 41             | 2010-01-01 00:00:00 |
+      | 19                | 41             | 2010-01-01 00:00:00 |
+      | 22                | 41             | 2010-01-01 00:00:00 |
+      | 23                | 41             | 2010-01-01 00:00:00 |
+
+  Scenario: Get breadcrumbs
```

Comment:
Maybe you should indicate that this is a list of non-necessarily parent-child relations in case a reader wonder if they must or not.

Reply:
They all are expired